### DISCUSSION TITLE
Image is not showing when using <Image /> but works correctly with <img />

### DISCUSSION THREAD

**Discussion by not4fame** [vercel/next.js#52619] [Help]
### Summary

I have image step_1.webp  stored in public folder.
This does not work:
`<Image
                src="/step_1.webp"
                alt="step one"
                width={270}
                height={143}
            />`

And this works correctly:
`<img
                src="/step_1.webp"
                alt="step one"
                width={270}
                height={143}
            />`

I also tried:
`import step_1 from '../../public/step_1.webp'
...
<Image
                src={step_1}
                alt="step one"
                width={270}
                height={143}
            />
`

### Additional information

```js
The image is also present in: next\.next\static\media\step_1.d9be8a23.webp
```

**Comment by icyJoseph** [comment]
How are you import `next/image`? 

What HTML does it render if you use `next/image`?

**Reply by not4fame** [reply]
Hi @icyJoseph here is rendered HTML for the component:

```
<img alt="step one of exchange" 
loading="lazy" 
width="270" 
height="143" 
decoding="async" 
data-nimg="1" 
style="color:transparent" 
srcset="/_next/image?url=%2F_next%2Fstatic%2Fmedia%2Fstep_1.d9be8a23.webp&amp;w=384&amp;q=75 1x, /_next/image?url=%2F_next%2Fstatic%2Fmedia%2Fstep_1.d9be8a23.webp&amp;w=640&amp;q=75 2x" 
src="/_next/image?url=%2F_next%2Fstatic%2Fmedia%2Fstep_1.d9be8a23.webp&amp;w=640&amp;q=75">
```
this is how I do the import `import Image from 'next/image'`
my next.js versions is 13.4.5

**Reply by not4fame** [reply]
checking the network I can see `"url" parameter is required` response for the `/_next/image?url=%2F_next%2Fstatic%2Fmedia%2Fstep_1.d9be8a23.webp&amp;w=640&amp;q=75` request

**Comment by not4fame** [comment]
I found the issue.
I am using NGINX to combine multiple services, the rule for next app was incorrect and it didn't pass the query parameters required by next/image
My incorrect rule was:
```
location ~ ^/(.*) {
        proxy_pass http://app/$1;
        proxy_http_version 1.1;
        proxy_set_header Upgrade $http_upgrade;
        proxy_set_header Connection "Upgrade";
    }
```
but it should be:
```
location ~ ^/(.*) {
        proxy_pass http://app/$1$is_args$args;
        proxy_http_version 1.1;
        proxy_set_header Upgrade $http_upgrade;
        proxy_set_header Connection "Upgrade";
    }
```

**Reply by icyJoseph** [reply]
Nice, yeah I was gonna suggest redirect/rewrite rules, or perhaps middleware, etc.
